types/hook.go

```diff
@@ -3,6 +3,7 @@ package types
 import (
 	"encoding/json"
 	"errors"
+	fmt "fmt"
```

Comment:
![giphy](https://user-images.githubusercontent.com/4105580/37838969-acf41d94-2e8f-11e8-8000-a6d170780296.gif)

Looks like we have a couple of them!

Reply:
Annoyingly, `goimports` started doing this out of nowhere. I'll look into it.